### PR TITLE
style(ui): container style adjustment

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Container level style adjustment

## Linear Issue Reference

log-2414

## Testing

test locally 
@logto-io/eng

### CODE REVIEW

File: packages/ui/src/scss/_desktop.scss [hunk 1]
```diff
@@ -54,10 +57,9 @@ $font-family: -apple-system,
   --font-title: 600 32px/40px #{$font-family};
   --font-title-medium: 600 28px/36px #{$font-family};
   --font-title-small: 600 24px/32px #{$font-family};
-  --font-body-bold: 500 16px/20px #{$font-family};
-  --font-body: 400 16px/20px #{$font-family};
-  --font-body-small: 500 14px/18px #{$font-family};
-  --font-caption: 500 14px/20px #{$font-family};
+  --font-body-bold: 500 14px/20px #{$font-family};
```
Comment (demonzoo) [inline packages/ui/src/scss/_desktop.scss:60]: Seems we do not have `body-bold` font in our design system.

File: packages/ui/src/scss/_desktop.scss [hunk 2]
```diff
@@ -54,10 +57,9 @@ $font-family: -apple-system,
...
+  --font-body: 400 14px/20px #{$font-family};
+  --font-caption: 400 14px/20px #{$font-family};
```
Comment (demonzoo) [inline packages/ui/src/scss/_desktop.scss:62]: These 2 also...

image

File: packages/ui/src/containers/SignInMethodsLink/index.module.scss [hunk 3]
```diff
@@ -1,6 +1,18 @@
...
+  width: 1px;
+  height: 12px;
+  position: absolute;
+  top: 50%;
+  right: 0;
+  transform: translateY(-50%);
+  background-color: var(--color-primary);
+  opacity: 60%;
+}
+
```
Comment (demonzoo) [inline packages/ui/src/containers/SignInMethodsLink/index.module.scss:15]: You can also put `&::after` in mixin

File: packages/ui/src/scss/_desktop.scss [hunk 4]
```diff
@@ -54,10 +57,9 @@ $font-family: -apple-system,
   --font-title: 600 32px/40px #{$font-family};
```
Comment (demonzoo) [inline packages/ui/src/scss/_desktop.scss:57]: I don't think we need a `fonts` mixin in each of the desktop / mobile stylesheets, as the fonts are considered universal and reusable across both platforms

**Review by demonzoo — APPROVED** [review]
Overall look good. Please check my comments before merging, nothing big deal though.

File: packages/ui/src/scss/_desktop.scss (hunk 4 above)
Comment (simeng-li) [reply packages/ui/src/scss/_desktop.scss:57]: nope is two different DS that's the case. And per our discussion is a mixed version not pure AC.

File: packages/ui/src/scss/_desktop.scss (hunk 1 above)
Comment (simeng-li) [reply packages/ui/src/scss/_desktop.scss:60]: yes, it is mapped with mobile design system for not. not entirely AC design system.

File: packages/ui/src/scss/_desktop.scss (hunk 2 above)
Comment (simeng-li) [reply packages/ui/src/scss/_desktop.scss:62]: ditto

**Pull request merged**